Dmf/Modules.Library/Dmf_UdeClient.md

```diff
@@ -403,10 +463,67 @@ NTSTATUS
 Parameter | Description
 ----|----
 DmfModule | An open DMF_UdeClient Module handle.
+UdecxUsbDevice | The device on which the endpoint is located.
 EndpointInit | An opaque structure that represent an initialization configuration for Endpoint.
 EndpointConfig | Endpoint configuration.
 Endpoint | A handle to a framework device object that represents the newly created Endpoint.
 
+-----------------------------------------------------------------------------------------------------------------------------------
+##### DMF_UdeClient_DeviceEndpointInformationGet
+
+````
+_IRQL_requires_max_(PASSIVE_LEVEL)
+VOID
+DMF_UdeClient_DeviceEndpointInformationGet(
+    _In_ DMFMODULE DmfModule,
+    _In_ UDECXUSBENDPOINT Endpoint,
+    _Out_opt_ UDECXUSBDEVICE* UdecxUsbDevice,
+    _Out_opt_ UCHAR* Address
+    )
+````
+
+Gets information about a given endpoint.
+
+##### Returns
+
+None
+
+##### Parameters
+Parameter | Description
+----|----
+DmfModule | An open DMF_UdeClient Module handle.
+Endpoint | The given endpoint.
+UdecxUsbDevice | The device to which the endpoint is attached.

```

Comment:
located?

Reply:
I don't see anything missing...

````
_IRQL_requires_max_(PASSIVE_LEVEL)
VOID
DMF_UdeClient_DeviceEndpointInformationGet(
    _In_ DMFMODULE DmfModule,
    _In_ UDECXUSBENDPOINT Endpoint,
    _Out_opt_ UDECXUSBDEVICE* UdecxUsbDevice,
    _Out_opt_ UCHAR* Address
    )
````

Gets information about a given endpoint.

##### Returns

None

##### Parameters
Parameter | Description
----|----
DmfModule | An open DMF_UdeClient Module handle.
Endpoint | The given endpoint.
UdecxUsbDevice | The device to which the endpoint is attached.
Address | The address of the endpoint.

Reply:
i understand now...fixed